src/test/java/org/cactoos/matchers/InputHasContentTest.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) for portions of project cactoos-matchers are held by
+ * Yegor Bugayenko, 2017-2018, as part of project cactoos.
+ * All other copyright for project cactoos-matchers are held by
+ * George Aristy, 2018.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.matchers;
+
+import org.cactoos.Input;
+import org.cactoos.io.InputOf;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.Test;
+import org.llorllale.cactoos.matchers.InputHasContent;
+
+/**
+ * Test case for {@link InputHasContent}.
+ *
+ * @author Vedran Vatavuk (123vgv@gmail.com)
+ * @version $Id$
+ * @since 1.1
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class InputHasContentTest {
+
+    @Test
+    public void matchesInputContent() throws Exception {
```

Comment:
@Vatavuk This test isn't working, it does not compares `String` and `InputOf` contents; this method should return `true` when using `new InputHasContent("Hello World!")`.

Reply:
@paulodamaso I'm not sure that I get what is the problem. I'm doing exactly as you described, comparing `Input` with `String`